src/Sylius/Bundle/CoreBundle/OAuth/UserProvider.php

```diff
@@ -85,7 +93,8 @@ public function __construct(
         RepositoryInterface $oauthRepository,
         ObjectManager $userManager,
         CanonicalizerInterface $canonicalizer,
-        CustomerRepositoryInterface $customerRepository
+        CustomerRepositoryInterface $customerRepository,
+        EventDispatcherInterface $eventDispatcher
```

Comment:
This is the main source of BC Break, if we will allow for null value here and check for it later, we can proceed and merge this PR to 1.7.

```suggestion
        ?EventDispatcherInterface $eventDispatcher = null
```